src/main/java/org/eclipse/yasson/internal/serializer/FloatArraySerializer.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2016, 2017 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2016, 2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Order of imports is not enough (and shall not be done) to update copyright year change. I would advise revert here.

Reply:
This class was modified in one of the commits. This change was later reverted, but it will remain in the history so it was modified in 2019.